asm/code_8010F28.s

```diff
@@ -29,7 +29,7 @@ sub_8010F28:
 	mov r8, r1
 	ldr r4, _0801108C
 	adds r0, r4
-	bl sub_80054BC
+	bl RunAxAnimationFrame_80054BC
```

Comment:
Any reason we are keeping the address in the names? If you are fairly confident on them, I say to just take the addresses out of the name but keep a comment with the address for documentation.